solver/kong/plugin.go

```diff
@@ -69,6 +69,9 @@ func (s *PluginCRUD) Update(arg ...crud.Arg) (crud.Arg, error) {
 	event := eventFromArg(arg[0])
 	plugin := pluginFromStuct(event)
 
+	if plugin.Service != nil {
+		plugin.Service = &kong.Service{ID: plugin.Service.ID}
+	}
```

Comment:
Could you please add this for the Route and Consumer case? Thanks!

Reply:
@hbagdi I just added it for `Route`, but `Consumer` doesn't seem to have a `Service`.

Reply:
Oh, I'm really sorry, I meant, we should add this for `plugin.Route` and `plugin.Consumer` as well.

Reply:
@hbagdi please review when you get a chance.. thanks!